db/postgres.go

```diff
@@ -218,6 +225,27 @@ func (p *PostgreSQL) ExtraIndexes(idxs []string) error {
 	if err := transform.ValidateIndexes(idxs); err != nil {
 		return fmt.Errorf("index name error: %w", err)
 	}
-	log.Output(1, fmt.Sprintf("indexes: %s", strings.Join(idxs, ", ")))
-	return fmt.Errorf("extra-indexes not implemented (yet)")
+	for _, idx := range idxs {
+		v := idx
+		name := "json."
+		typeIdx := "root"
+		nameIdx := fmt.Sprintf("%s%s", name, v)
+		valueIdx := idx
+		if strings.Contains(idx, ".") {
+			typeIdx = "no_root"
+			nameIdx = fmt.Sprintf("%s%s", name, v)
+			valueIdx = v
+		}
+		p.ExtraIndexesFields.Type = typeIdx
+		p.ExtraIndexesFields.Name = nameIdx
+		p.ExtraIndexesFields.Value = valueIdx
+		p.loadTemplates()
```

Comment:
Isso já acontece no `NewPostgreSQL`. Pq precisamos disso aqui também?

Reply:
vc diz o p.loadTemplates()? se for coloquei ele ai porque voce havia dito que precisava rodar o loadTemplates ( naquela PR q vc fechou ) e de fato depois q coloquei comecou a funcionar.